src/main/java/library/app/spring/dao/impl/UserDaoImp.java

```diff
@@ -23,7 +23,8 @@ public void add(User user) {
     @Override
     public List<User> listUsers() {
         @SuppressWarnings("unchecked")
```

Comment:
try to remove `SuppressWarnings`. Bohdan wrote how to do it in slack

Reply:
done